src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingLocationMapping.java

```diff
@@ -26,18 +25,15 @@ public static List<HearingLocations> getHearingLocations(SscsCaseData caseData,
         VenueDetails venueDetails = referenceData.getVenueDataLoader().getVenueDetailsMap()
             .get(String.valueOf(venueId));
 
-        if (venueDetails != null) {
-            epimsId = venueDetails.getEpimsId();
-        }
+        String epimsId = Optional.ofNullable(venueDetails)
```

Comment:
I guess it's a compromise lol.. could maybe just use a ternary operator on line 33 if @nickwest1 is happy with that?

Reply:
I prefer what Del's done TBH